vignettes/Model.Rmd

```diff
@@ -1,5 +1,7 @@
 ---
 title: "Model Structure"
+header-includes:
+  -\usepackage{svg}
```

Comment:
I am getting an error when I try to render that package "svglite" is required. Can the vignette function without it as a dependency?

Reply:
I had all kinds of problems working out how to get an svg file to plot. It looks like mine doesn't need the svg package to plot after all... When you say render do you mean knitting through R or do you mean something else?

Reply:
I had a similar issue. Is there a reason why you've choosen svg?
I believe .png would be the standard format.

Reply:
I think this works now